openshift_images/samples-operator-alt-registry.adoc

```diff
@@ -14,8 +14,14 @@ include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
 
 //Incorrect rendering of ROSA attributes discovered during Pruning HCP task. I believe some of the following content is not correct for managed OpenShift. I have created JIRA issue to investigate further.
 
-include::modules/cli-installing-cli.adoc[leveloffset=+2]
-//the CLI module will need to be made compliant in a separate PR; touches too many files (SD, during CQA 2.0)
+// Installing the OpenShift CLI on Linux
+include::modules/cli-installing-cli-linux.adoc[leveloffset=+1]
+
+// Installing the OpenShift CLI on Windows
+include::modules/cli-installing-cli-windows.adoc[leveloffset=+1]
+
+// Installing the OpenShift CLI on macOS
+include::modules/cli-installing-cli-macos.adoc[leveloffset=+1]
```

Comment:
Looks like these should be `[leveloffset=+2]` like the module they replaced

Reply:
:heavy_check_mark: